utility_network/display-utility-associations/README.metadata.json

```
@@ -0,0 +1,26 @@
+{
+  "category": "Utility network",
+  "description": "Create graphics for utility associations in a utility network.",
+  "ignore": false,
+  "images": [
+    "DisplayUtilityAssociations.png"
+  ],
+  "keywords": [
+    "associating",
+    "association",
+    "attachment",
+    "connectivity",
+    "containment",
+    "relationships"
+  ],
```

this still needs updated to include the relevant APIs:
```suggestion
  "keywords": [
    "associating",
    "association",
    "attachment",
    "connectivity",
    "containment",
    "relationships",
    "GraphicsOverlay",
    "UtilityAssociation",
    "UtilityAssociationType",
    "UtilityNetwork"
  ],
```